### ISSUE TITLE
training on new data/change num_classes in mambaBCD

### ISSUE THREAD

**Issue by ZhiyiHe1997** [ChenHongruixuan/ChangeMamba#116]
hello, is the num_classes fixed in MambaBCD, I changed it to 3 or more classes, the last layer keep output '_**(main_clf): Conv2d(128, 2, kernel_size=(1, 1), stride=(1, 1)_**'  . 
I also find a interesting thing, I disable the **class ChangeDetectionDatset(Dataset)** in _**make_data_loader.py**_ ,  the **train_MambaBCD.py** can still training.
Can you help me to explain these? Thanks.

**Comment by ZhiyiHe1997** [comment]
train_data_loader = DataLoader(dataset, batch_size=args.batch_size, shuffle=args.shuffle, num_workers=4)
in train_MambaBCD.py is not used

**Comment by makovez** [comment]
did you solve the issue of multi class output ?
